.circleci/config.yml

```diff
@@ -231,3 +267,19 @@ workflows:
             branches:
               only:
                 - main
+      - run-all-nuts:
+          filters:
+            branches:
+              only:
+                - main
+
+  monitor-all-nuts:
+    when: << pipeline.parameters.run-just-nuts >>
```

Comment:
The condition that prevents this workflow from running under typical conditions, `run-just-nuts` is false by default.